README.md

```diff
@@ -430,6 +430,46 @@ public function getUser($credentials, UserProviderInterface $userProvider)
 The logged-in user will be an instance of `KnpU\OAuth2ClientBundle\Security\User\OAuthUser` and will
 have the roles `ROLE_USER` and `ROLE_OAUTH_USER`.
 
+## Storing and refreshing tokens
+
+You have a couple of options to store access tokens for use at a later time:
+
+1. Store the `AccessToken` object (eg. serializing into the session), this allows you to check expiry before refreshing
+    ```php
+    // Fetch and store the AccessToken
+    $accessToken = $client->getAccessToken();
+    $session->set('access_token', $accessToken);
+
+    // Load the access token from the session, and refresh if required
+    $accessToken = $session->get('access_token');
+
+    if ($accessToken->hasExpired()) {
+        $accessToken = $client->refreshAccessToken($accessToken);
+
+        // Update the stored access token for next time
+        $session->set('access_token', $accessToken);
+    }
+    ```
+
+2. Store just the refresh token string (eg. in the dabatase `user.refresh_token`), this means you must always refresh
+    ```php
+    // Fetch the AccessToken and store the refresh token
+    $accessToken = $client->getAccessToken();
+    $user->setRefreshToken($accessToken->getRefreshToken());
+    $entityManager->flush();
```

Comment:
You could ALSO store the `access_token` and expires in the database if you wanted to. Maybe we could make a note of that?

```php
// you can also store the access token and expiration and then avoid the refresh
// until the access token is actually expired
```